llm_bioinformatics_research/src/Database/Connection.js

```diff
@@ -0,0 +1,61 @@
+require('dotenv').config();
+const express = require('express');
+const { MongoClient } = require('mongodb');
+const bcrypt = require('bcrypt');
+const cors = require('cors');
+
+const app = express();
+const port = 5000;
+const saltRounds = 10;
+
+app.use(cors());  
+app.use(express.json());  
+
+const uri = process.env.MONGO_URI;
+const client = new MongoClient(uri, { useNewUrlParser: true, useUnifiedTopology: true });
+
+async function connectDB() {
+  try {
+    await client.connect();
+    console.log("Connected to MongoDB");
+  } catch (error) {
+    console.error("Error connecting to MongoDB", error);
+  }
+}
+connectDB();
+
+app.post('/signup', async (req, res) => {
+  try {
+    const { user_name, email, password } = req.body; 
```

Comment:
This would have to be a change on my end too but I think getting the user's first and last name during the signup phase could be good, and then we can save that information into the database as well. This can help with populating the profile page with more information!